test/unit/theory/theory_strings_rewriter_white.h

```diff
@@ -866,6 +866,14 @@ class TheoryStringsRewriterWhite : public CxxTest::TestSuite
                        d_nm->mkNode(kind::STRING_STRREPL, x, a, empty),
                        a);
     sameNormalForm(lhs, rhs);
+
+    {
+      // (str.contains (str.++ x "A") (str.++ x "B")) ---> false
```

Comment:
FWIW, this one should be doable with strip constant endpoints.

Reply:
Great point, I changed it to `(str.contains (str.++ x "A") (str.++ "B" x)) ---> false`